react/features/video-quality/components/VideoQualityLabel.web.js

```diff
@@ -1,22 +1,46 @@
+// @flow
+
 import Tooltip from '@atlaskit/tooltip';
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
 
 import { translate } from '../../base/i18n';
 import { CircularLabel } from '../../base/label';
 import { MEDIA_TYPE } from '../../base/media';
 import { getTrackByMediaTypeAndParticipant } from '../../base/tracks';
 
+import AbstractVideoQualityLabel, {
+    _abstractMapStateToProps,
+    type Props as AbstractProps
+} from './AbstractVideoQualityLabel';
+
+type Props = AbstractProps & {
+
+    /**
+     * The message to show within the label.
+     */
+    _labelKey: string,
+
+    /**
+     * The message to show within the label's tooltip.
+     */
+    _tooltipKey: string,
+
+    /**
+     * The redux representation of the JitsiTrack displayed on large video.
+     */
+    _videoTrack: Object
+};
+
 /**
  * A map of video resolution (number) to translation key.
  *
  * @type {Object}
  */
 const RESOLUTION_TO_TRANSLATION_KEY = {
-    720: 'videoStatus.hd',
-    360: 'videoStatus.sd',
-    180: 'videoStatus.ld'
+    '720': 'videoStatus.hd',
```

Comment:
why the change?

also, come tho think of it, the computation for RESOLUTIONS below can be simplified.

Reply:
This file was not flow-ified before and now that I added the annotation, it required me to change it.

I agree with simplification, but I think this is not part of this PR - Although I'm working on a larger refactor for this component and I'll include that simplification into that PR for sure

Reply:
It's one line though.

Reply:
Ok, shoot!